Exception/WeirdException.php

```diff
@@ -6,4 +6,5 @@ class WeirdException extends \Exception
 {
     const ERROR_DEV_MAPPER = 'Error response from daemon: open /dev/mapper/';
     const ERROR_DEVICE_RESUME = 'Error response from daemon: devicemapper: Error running deviceResume dm_task_run failed';
+    const ERROR_DEV_MAPPER_BUILD = '[0m / open /dev/mapper/';
```

Comment:
Jo, je to escapovací sekvence na zrušení všech barev. Podle mě to bude v terminálu, kterej to má vypnutý, bez tý sekvence.